### ISSUE TITLE
Issue from Niels

### ISSUE THREAD

**Issue by jmaassen** [research-software-directory/RSD-as-a-service#1324]
We received the following from @nielsdrost in the old RSD repo (see: https://github.com/research-software-directory/research-software-directory/issues/853)

Sometimes I find a clearly out of date page (e.g. a project that has finished with no results).

Rather than pointing this out to the (co) maintainer of the page it would be nice to be able to somehow "flag" that this page is out of date, perhaps a bit like the wikipedia "issues with this page" markers. It's fine if I can only do this when I'm allowed to edit the page.

This would perhaps even allow a report of problematic pages and/or someway to contact the maintainer automatically.

**Comment by jmaassen** [comment]
Indeed, we would like to detect outdated or incomplete pages somehow, not just for projects, but also software. The  "issues with this page" approach is an interesting idea.

We do need to thing a bit about what "outdated" or "incomplete" would mean exactly.

**Comment by bakhshir** [comment]
Maybe like review request, that sends automatic message to the main contact point? That would also solve the other issue that sometimes the contact points never got maintainer links.
